eventstore/dynamodb/eventstore.go

```diff
@@ -65,14 +69,26 @@ func (c *EventStoreConfig) provideDefaults() {
 	if c.Region == "" {
 		c.Region = "us-east-1"
 	}
+	if c.Endpoint == "" {
+		c.Endpoint = "http://localhost:8000"
+	}
+	if c.Username == "" {
+		c.Username = "doesnotmatter"
+	}
+	if c.Password == "" {
+		c.Password = "doesnotmatter"
+	}
 }
 
 // NewEventStore creates a new EventStore.
 func NewEventStore(config *EventStoreConfig) (*EventStore, error) {
 	config.provideDefaults()
 
+	creds := credentials.NewStaticCredentials(config.Username, config.Password, "")
```

Comment:
I would like to only use external credentials, that is from the confit or env vars. It minimizes setup and is the suggested way of using the AWS SDK as far as I know.